README.md

```diff
@@ -158,70 +167,80 @@ const flipFlop = timer(0, 1000).pipe(
 ```
 
 With `switchMap` we can map an observable to a component observable depending on a condition, or to an observable emitting one of either: `null | undefined | false` to remove it from the DOM.
-```typescript
+
+```jsx
 import { switchMap } from 'rxjs/operators';
 import { of } from 'rxjs';
 
-const ConditionalComponentsExample = Div(
-  flipFlop.pipe(
-    switchMap(visible => visible ? Div`Now you see me!` : of(null)),
-  ),
-  // Or more simply as: conditional(flipFlop, Div`Now you see me!`), using the 'conditional' helper function from rxfm.
-);
+const ConditionalComponentsExample = () => <div>
+  {flipFlop.pipe(
+    switchMap(visible => visible ? <div>Now you see me!</div> : of(null)),
+  )}
+</div>;
 ```
 
-You may also be tempted to use `switchMap` to transform and array observable into an array of components (similar to using Array.map in React), but this will be rather inefficient as the components will be recreated each time the observable emits. The `mapToComponents` operator function should be used instead in this case as this will ensure that components are only recreated when necessary (see the "Dynamic Component Arrays" section).
+This may also be written more simply as: `conditional(flipFlop, <div>Now you see me!</div>)` using the `conditional` helper function from rxfm.
```

Comment:
```suggestion
This may also be written more simply as below using the `conditional` helper function from rxfm:

```jsx
conditional(flipFlop, <div>Now you see me!</div>)
```
```